microreactors/gcmr/core/Neutronics/Griffin_steady_state.i

```diff
@@ -0,0 +1,164 @@
+######################################################################################################
+## Griffin steady state Model of whole-core Gas-cooled Microreactor
+# If using or referring to this model, please cite as explained in
+# https://mooseframework.inl.gov/virtual_test_bed/citing.html
+#####################################################################################################
+
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = '../MESH/Griffin_mesh_in.e'
+  []
+
+  [fmg_id]
+    type = SubdomainExtraElementIDGenerator
+    input = fmg
+    subdomains = '10 100 101 102 103 200 201 400 401 4000 4001 40000 40001 300 301 600 602 603 604 1000 1003 19000 29000 39000 49000 59000 19003 29003 39003 49003 59003 19900 29900 39900 49900 59900 19903 29903 39903 49903 59903 1777 1773 250'
+    extra_element_id_names = 'material_id'
+    extra_element_ids = '803 802 802 810 806 807 807 700 700 701 701 702 702 807 807 811 811 809 807 805 805 764 764 764 764 764 764 764 764 764 764 754 754 754 754 754 754 754 754 754 754 809 809 811'
```

Comment:
Can we make this block ID numbering to match the mapping of cross section IDs? It's really difficult to follow the mesh-material mapping with all of these numbers

Reply:
Thank you for your suggestion. I think it can be followed by examining the rename_blocks in the mesh input. But we can improve later too.

Reply:
I would push for using the names in Mesh/fmg_id/subdomains here then. @GiudGiud is there any standard that VTB enforces to make mesh block / material ID mapping clear to the user?